src/v2/Components/CollectionsHubsNav.tsx

```diff
@@ -29,7 +29,11 @@ export const CollectionsHubsNav: FC<CollectionsHubsNavProps> = props => {
           to={`/collection/${hub.slug}`}
           src={resize(hub.thumbnail, { width: 357, height: 175 })}
           ratio={[0.49]}
-          title={<Serif size="4t">{hub.title}</Serif>}
+          title={
+            <Text variant="text" px={1}>
```

Comment:
Oh, this `px={1}` is added here to keep the text from wrapping in a weird location. "Impressionist & Modern Art" was wrapping between "Modern" and "Art" after I updated the font. I don't think these 6 hub names are changing any time soon, so adding some padding seemed like an okay way to make it wrap between "&" and "Modern".